src/test/java/com/wikia/webdriver/testcases/createawikitests/CreateWikiTests_lang.java

```diff
@@ -15,9 +15,6 @@
 
 import org.testng.annotations.Test;
 
-/**
- * @author Karol 'kkarolk' Kujawiak 1. Create wiki in different languages
- */
 @Test(groups = {"CNW_lang"})
 public class CreateWikiTests_lang extends NewTestTemplate {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00101)